CPAC/pipeline/engine.py

```diff
@@ -1984,13 +2010,28 @@ def ingress_freesurfer(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id):
         creds_path=data_paths["creds_path"],
         dl_dir=cfg.pipeline_setup["working_directory"]["path"],
     )
+    node = fs_ingress
+    out = "outputspec.data"
+    node_name = "freesurfer_config_ingress"
+
+    if fs_path.endswith(".nii.gz" or ".nii"):
+        reorient = pe.Node(
+            interface=afni.Resample(),
+            name=f"reorient_fs_{part_id}_{ses_id}",
+        )
+        reorient.inputs.orientation = cfg.pipeline_setup["desired_orientation"]
+        reorient.inputs.outputtype = "NIFTI_GZ"
+        wf.connect(fs_ingress, "outputspec.data", reorient, "in_file")
+        node = reorient
+        out = "out_file"
+        node_name = "reorient_fs"
```

Comment:
If we do this, then this can just be

```suggestion
    if fs_path.endswith(".nii.gz" or ".nii"):
        reorient = rpool.reorient_node("freesurfer-subject-dir",  "freesurfer_config_ingress", f"{part_id}_{ses_id}")
        wf.connect(fs_ingress, "outputspec.data", reorient, "in_file")
```

and the `rpool.set_data` below can go back to what it was, but in an `else:` block:

```Python
    else:
        rpool.set_data(
            "freesurfer-subject-dir",
            fs_ingress,
            "outputspec.data",
            {},
            "",
            "freesurfer_config_ingress",
        )
```

Reply:
with the changes, I think this is not needed.